integration-tests/tests/tsconfig.common.json

```diff
@@ -3,7 +3,7 @@
 		"composite": true,
 		"target": "es2022",
 		"module": "es2022",
-		"moduleResolution": "node",
+		"moduleResolution": "Bundler",
```

Comment:
Did you check this works with `npm test --workspace @realm/integration-tests` (where we use raw Mocha and `tsx` to transpile TypeScript)?

Reply:
Works ✅